ci-operator/step-registry/openshift-qe/installer/bm/day2/odf/openshift-qe-installer-bm-day2-odf-commands.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+
+if [ ${BAREMETAL} == "true" ]; then
+  bastion="$(cat /bm/address)"
+  # Copy over the kubeconfig
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion "cat ~/mno/kubeconfig" > /tmp/kubeconfig
+  # Setup socks proxy
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion -fNT -D 12345
+  export KUBECONFIG=/tmp/kubeconfig
+  export https_proxy=socks5://localhost:12345
+  export http_proxy=socks5://localhost:12345
+  oc --kubeconfig=/tmp/kubeconfig config set-cluster bm --proxy-url=socks5://localhost:12345
+fi
+
+oc config view
+oc projects
```

Comment:
Remove

Reply:
Why? We seem to have this in all the steps, i.e.: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh#L7-L8
Or do you mean to keep it only in performance related ones?